### DISCUSSION TITLE
Thank you. Solid UI library

### DISCUSSION THREAD

**Discussion by amirvahidian** [mdbootstrap/TW-Elements#1857] [Kind words & general feedback]
I have been using it for a Django project I am working on and it is mostly working well without much issue. Implementation of elements in forms did cause some problems but I found a way around that issue and all in all the upside is well worth having to deal with a few problems here and there

**Comment by filipkappa** [comment]
Thanks so much!

The project is still in beta, so that's to be expected, but a little bird told me that a stable release is coming soon 🤫 

I hope we will be able to solve most of the issues you had with TE - just make sure to post them on Issues or Support! 😉
